src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1314,8 +1314,12 @@ export default class AircraftInstanceModel {
         const isTimeToStartTurning = distanceToWaypoint < nm(calculateTurnInitiaionDistance(this, waypointPosition));
         const closeToBeingOverFix = distanceToWaypoint < PERFORMANCE.MAXIMUM_DISTANCE_TO_PASS_WAYPOINT_NM;
         const closeEnoughToFlyByFix = distanceToWaypoint < PERFORMANCE.MAXIMUM_DISTANCE_TO_FLY_BY_WAYPOINT_NM;
-        // TODO: abstract this logic to helper method
-        const shouldMoveToNextFix = closeToBeingOverFix || (closeEnoughToFlyByFix && isTimeToStartTurning);
+        const wantToFlyByFix = closeEnoughToFlyByFix && isTimeToStartTurning;
```

Comment:
well, in this sense, _should_ is used as in 'this should happen now'.  So yeah, I still prefer `shouldFlyByFix`.  

but you're so much better with words than I am 😉